src/undomanager.js

```diff
@@ -7,7 +7,7 @@ define([
     $,
     _,
     util,
-    UNDO_ALERT
+    undo_alert
```

Comment:
I think the convention is lowercase except for test XML. Possibly uppercase for static strings, which is what this was before, although I think this was the first place I saw that.